samples/java_sample/java_modules/RemotePrinter/pom.xml

```diff
@@ -0,0 +1,41 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.microsoft.azure.gateway</groupId>
+  <artifactId>sample-printer-module-remote</artifactId>
+  <version>1.0.0</version>
```

Comment:
Since this is a sample I'd prefer we mark it with a pre-release version (e.g. 0.0.1).